lib/reform/form/dry.rb

```diff
@@ -1,97 +1,19 @@
 require "dry-validation"
+require "dry/validation/version"
 require "reform/validation"
+require "reform/form/dry/input_hash"
 
 module Reform::Form::Dry
-  class Schema < Dry::Validation::Schema
-  end
-
   def self.included(includer)
-    includer.send :include, Validations
-    includer.extend Validations::ClassMethods
-  end
-
-  module Validations
-    module ClassMethods
-      def validation_group_class
-        Group
-      end
-    end
-
-    def self.included(includer)
-      includer.extend(ClassMethods)
+    if Gem::Version.new(Dry::Validation::VERSION) > Gem::Version.new("0.13.3")
+      require "reform/form/dry/new_api"
+      validations = Reform::Form::Dry::NewApi::Validations
+    else
```

Comment:
Shouldn't we just require Dry >= 1.0 in 2.3?

Reply:
I'm quite sure there are a lot of people with 2.3.0.rcx and dry-v 0.13 so I think we should support both and only 1.x for reform 3.x
What do you think?

Reply:
Since we're throwing away most of Reform 2.x soon, yes, you're right! :pray:

Reply:
Still in draft here so we can discuss a better solution maybe to require files for a specific file version (same for the tests) but yeah for now I want to have it working and then I can improve.
That is why the PR is still in DRAFT :+1: